client/landing/stepper/declarative-flow/internals/steps-repository/woo-confirm/index.tsx

```diff
@@ -119,24 +102,18 @@ const WooConfirm: Step = function WooCommerceConfirm( { navigation } ) {
 
 	// Filter the Woop transferring blockers.
 	const transferringBlockers = eligibilityHolds?.filter(
-		( hold: TransferEligibilityError ) => ! TRANSFERRING_NOT_BLOCKERS.includes( hold.code )
+		( hold: TransferEligibilityHold ) => ! TRANSFERRING_NOT_BLOCKERS.includes( hold )
 	);
 
 	const isTransferStuck = latestAtomicTransfer?.is_stuck || false;
 	const isBlockByTransferStatus = latestAtomicTransferError || false;
 
 	// Add blocked-transfer-hold when something is wrong in the transfer status.
 	if (
-		! transferringBlockers?.includes( {
-			code: eligibilityHoldsConstants.BLOCKED_ATOMIC_TRANSFER,
-			message: '',
-		} ) &&
+		! transferringBlockers?.includes( 'BLOCKED_ATOMIC_TRANSFER' ) &&
```

Comment:
Typescript typechecks this now that `transferringBlockers` has specific types. As a result, `includes` can only be passed values of the `statusMapping` object. I think this is enough to avoid needing the constant here. (If we change it to something random, there'd be a typescript error.) 

But the side problem here is that there are two identical lists of constants stored in different places 🙃